### ISSUE TITLE
Getting ImportError

### ISSUE THREAD

**Issue by cuanim** [dbr/tvnamer#200]
I am using tvnamer on Windows 10 and I installed it using pip. When I ran it, this was the output

C:\Users\HP>tvnamer --help
Traceback (most recent call last):
  File "c:\python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "c:\python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python39\Scripts\tvnamer.exe\__main__.py", line 4, in <module>
  File "c:\python39\lib\site-packages\tvnamer\main.py", line 18, in <module>
    import tvdb_api
  File "c:\python39\lib\site-packages\tvdb_api.py", line 35, in <module>
    from requests_cache.backends.base import _to_bytes, _DEFAULT_HEADERS
ImportError: cannot import name '_to_bytes' from 'requests_cache.backends.base' (c:\python39\lib\site-packages\requests_cache\backends\base.py)

I have tried uninstalling and installing it again but the problem persist. Any idea why is this happening?
Also if it helps, I got this warning while installing

WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>   
distutils: c:\python39\Include\UNKNOWN   
sysconfig: c:\python39\Include   
WARNING: Additional context:   
user = False    
home = None   
root = None   
prefix = None

**Comment by gene1wood** [comment]
This was reported in dbr/tvdb_api#92 and there's a PR with a fix in dbr/tvdb_api#93

In the meantime you can workaround this by downgrading `requests-cache` to `0.5.2`. This can be done by running

```
pip install -U "requests-cache<0.6.0"
```

**Comment by dbr** [comment]
I've just released tvdb_api 3.1.0 which fixes this for now - new installs should pick this up automatically, or for currently broken installs the following should work:

```
pip install -U "tvdb_api"
```
